template/CHART_NAME/templates/daemonset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   {{- if .Values.%%MAIN_OBJECT_BLOCK%%.updateStrategy }}
   updateStrategy: {{- toYaml .Values.%%MAIN_OBJECT_BLOCK%%.updateStrategy | nindent 4 }}
   {{- end }}
-  {{- $podLabels := include "common.tplvalues.merge" (dict "values" .Values.%%MAIN_OBJECT_BLOCK%%.podLabels .Values.commonLabels "context" .) | fromYaml }}
+  {{- $podLabels := include "common.tplvalues.merge" (dict "values" (list .Values.%%MAIN_OBJECT_BLOCK%%.podLabels .Values.commonLabels) "context" .) | fromYaml }}
```

Comment:
Good catch and thanks for the PR @maxnitze! Could you also remove the `fromYaml` part from `template/` files? This was actually included in the `common.tplvalues.merge` [template](https://github.com/bitnami/charts/blob/a5c43a1c7a342dc39ba4d08d9aaced81b3228b54/bitnami/common/templates/_tplvalues.tpl#L35) so it's not necessary here.

Reply:
Hey @FraPazGal , I'm pretty sure this is still necessary. In [line 37](https://github.com/bitnami/charts/blob/a5c43a1c7a342dc39ba4d08d9aaced81b3228b54/bitnami/common/templates/_tplvalues.tpl#L37) the `$dst` dictionary is rendered to yaml again (since that is the only way the templates can pass on things).

Reply:
You are right about the `toYaml` conversion at the end, I missed it. That said, the `fromYaml` doesn't maje any difference when rendering the template.

I made a couple of tests using some of our chart which do not implement the `fromYaml` and I haven't seen issues. This is probably because it doesn't matter whether the value is a string or an object (as you are just printing it). wdyt?

Reply:
Hm. Yeah, you might be right. Did not check all the places, but I think it should wirk either way.

I personally like working with "structures" better. But in this case both solutions are ok for me, I think. It's a constant "fromYaml-toYaml" with the common helpers anyways :)

If it's ok for you I would keep it as it is. If not, I can change it. Your call :)

Reply:
Yes I agree with your reasoning. I have checked our charts and it seems the vast majority (except a few instances in `bintami/minio` and `bitnami/mlflow`) is using the `common.tplvalues.merge` without the `fromYaml`. Apologies for being so nit-picky here but I think we should remove it just to keep consistency with the rest of charts.